### PR TITLE
Rename grafana dashboard state

### PR DESCRIPTION
**Pull request by anlutro**
Consistency with other state types. I hope to contribute a `grafana2_datasource` state soon.

### CODE REVIEW

**Comment by terminalmage** [comment]
NOTE: As this is in develop only, the renaming should be fine as it does not affect an actual SaltStack release.

**Comment by terminalmage** [comment]
@anlutro Did you mean ``grafana_datasource``? I noticed that you renamed this to ``grafana_dashboard``, not ``grafana2_dashboard``. Just want to make sure the naming is consistent.

**Comment by anlutro** [comment]
@terminalmage Sorry, I was unclear. I'm working on a separate pull request for a `grafana2_datasource` state module, which is what prompted me to make this pull request.

When you mention it, the state I'm renaming here originally was named `grafana2` to avoid conflicts with the existing `grafana` state, but if it's to be named `grafana_dashboard` instead, I can probably drop the `2` from the filename as well, right?

**Comment by terminalmage** [comment]
@anlutro If you think that would reduce confusion, sure.

File: salt/states/grafana2_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+import requests
+
+from salt.ext.six import string_types
+from salt.utils.dictdiffer import DictDiffer
+
+
+def __virtual__():
+    """Only load if grafana v2.0 is configured."""
+    if __salt__['config.get']('grafana_version', 1) == 2:
+        return __virtualname__
```
Comment (cachedout) [inline salt/states/grafana2_dashboard.py]: We need to define __virtualname__ first. ;]

File: salt/states/grafana2_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,547 @@
...
+import json
+
+import requests
+
+from salt.ext.six import string_types
+from salt.utils.dictdiffer import DictDiffer
+
+
+def __virtual__():
+    """Only load if grafana v2.0 is configured."""
```
Comment (cachedout) [inline salt/states/grafana2_dashboard.py]: As a style rule, we prefer single-quotes for docs strings, please.

**Comment by cachedout** [comment]
Hi @anlutro Just a couple of small changes needed here and we'll be set. Please have a look and let me know what you think. Thanks!

**Comment by anlutro** [comment]
@cachedout Changes should be adressed. Thanks for the feedback!

**Comment by cachedout** [comment]
Looks great! Thanks for this, @anlutro. Much appeciated.

**Pull request merged**